beluga_amcl/config/Amcl.cfg

```diff
@@ -263,6 +263,12 @@ gen.add(
     default=0.05, min=0., max=1.
 )
 
+gen.add(
+    "model_unknown_space", bool_t, 0,
+    "Whether to model unknown space or assume it free.",
+    default=False,
+)
```

Comment:
@LaBruma this is stacked on top of #448, isn't it? If so, we should target that branch instead of `main`.